Apps/CommonData/src/Utils/DateFormatter.cs

```diff
@@ -13,128 +13,131 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 // -------------------------------------------------------------------------
-namespace HealthGateway.Common.Data.Utils;
```

Comment:
We allow both styles of namespace definitions and I prefer the file-scoped namespace for new files, so I'd recommend leaving this as-is.  Without the extra noise, it will make it easier to tell if there are any other changes to this file in this PR.